manim/animation/animation.py

```diff
@@ -22,6 +22,68 @@
 
 
 class Animation:
+    """An animation.
+
+    Animations have a fixed time span.
+
+    Parameters
+    ----------
+    mobject
+        The mobject to be animated. This is not required for all types of animations.
+    lag_ratio
+        Defines the delay after which the animation is applied to submobjects. This lag
+        is relative to the duration of the animation.
+
+        This does not influence the total
+        runtime of the animation. Instead the runtime of individual animations is
+        adjusted so that the complete animation has the defined run time.
+
+    run_time
+        The duration of the animation in seconds.
+    rate_func
+        The function defining the animation progress based on the relative runtime.
+
+        For example ``rate_func(0.5)`` is the proportion of the animation that is done
+        after half of the animations run time.
+    name
+        The name of the animation. This gets displayed while rendering the animation.
+        Defualts to <class-name>(<Mobject-name>).
+    remover
+        Whether the given mobject should be removed from the scene after this animation.
+    suspend_mobject_updating
+        Whether updaters of the mobject should be suspended during the animation.
+
+
+    Examples
+    --------
+
+    .. manim:: LagRatios
+
+        class LagRatios(Scene):
+            def construct(self):
+                ratios = [0, 0.1, 0.5, 1, 2]  # demonstrated lag_ratios
+
+                # Create dot groups
+                group = VGroup(*[Dot() for _ in range(4)]).arrange_submobjects()
```

Comment:
```suggestion
                group = VGroup()
                for _ in range(0,4):
                    group.add(Dot())
                group.arrange(RIGHT)
```
Zen of python:
Sparse is better than dense.
Readability counts.

Reply:
In my projects, I also really like to use this long and dense code lines:
VGroup(*[Dot() for _ in range(4)]).arrange_submobjects().
But I think for the docs, shorter and more lines is better.
However, this is free to be discussed a bit more.

Reply:
If this was a tutorial example on how to use manim I would agree. But in the examples I don't want to have most of the code dedicated to setup unrelated to the feature that should be demonstrated. I also added a comment explaining what the code does. Imo that should be enough.

Reply:
okay, then I am fine with that!